lcm/lcm.go

```diff
@@ -24,26 +24,6 @@ type Controller interface {
 }
 
 type CRStatus struct {
-	Phase      Phase      `json:"phase"`
-	Properties Properties `json:"properties"`
+	Condition  v1.HarborClusterCondition `json:"condition"`
```

Comment:
`HarborClusterConditionType` is only defined ready status, maybe we should add other states, e.p: Pending, Creating?

Reply:
Pending, Creating statuses are included in the Ready Status `False`. we can describe detail info in `Message` or  `Reason` field.